test/tests/transfers/multiapp_reporter_transfer/dist_vector/tests

```diff
@@ -0,0 +1,49 @@
+[Tests]
+    issues = '#26953'
+    design = 'MultiAppReporterTransfer.md'
+    [vec_rep]
+        requirement = "The system shall support the ability to transfer reporter
+        data in a N subapp to 1 parent app, and 1 parent app to N subapp fashion when"
+        # We only allow recover testing on a single example. During recover
+        # testing checkpoints are overwritten and used by the wrong test leading
+        # to an incorrect test failure.
+      [reporter_transfer]
+        type = 'JSONDiff'
+        input = 'main.i'
+        jsondiff = 'main_out.json main_out_sub0_out.json  main_out_sub1_out.json main_out_sub2_out.json  main_out_sub3_out.json'
+        detail = "the number of procs per app is default,"
+      []
+      [reporter_transfer_min_procs_2]
+        type = 'JSONDiff'
+        cli_args = "MultiApps/sub/min_procs_per_app=2 Outputs/file_base=main_2_out"
+        input = 'main.i'
+        jsondiff = 'main_2_out.json main_2_out_sub0_out.json main_2_out_sub1_out.json main_2_out_sub2_out.json main_2_out_sub3_out.json'
+        min_parallel = 2
+        detail = "the number of procs per app is less than the number of apps,"
+      []
+      [reporter_transfer_min_procs_5]
+        type = 'JSONDiff'
+        cli_args = "MultiApps/sub/min_procs_per_app=5 Outputs/file_base=main_5_out"
+        input = 'main.i'
+        jsondiff = 'main_5_out.json main_5_out_sub0_out.json  main_5_out_sub1_out.json main_5_out_sub2_out.json  main_5_out_sub3_out.json'
+        min_parallel = 5
+        detail = "the number of procs per app is larger than the number of apps,"
+      []
+      [reporter_send_error]
+        type = 'RunException'
+        input = 'main_dist.i'
+        expect_err = 'The Reporter value "main/dis_vec" is being produced in DISTRIBUTED mode, but the MultiAppReporterTransfer "to_sub" is requesting to consume it in REPLICATED mode, which is not supported.'
+        allow_test_objects=true
+        detail = "but not when the reporter being tranferred is distributed in the main app,"
```

Comment:
instead of doing this, create a new nested tests block and have the base requirement be : the system shall error if